docs/migration_guides/opflow_migration.rst

```diff
@@ -0,0 +1,1748 @@
+       ┌────────────────┐
+    q: ┤ U3(2,-π/2,π/2) ├
+       └────────────────┘
+
+**Alternative**
+
+.. code-block:: python
+
+    >>> from qiskit.quantum_info import SparsePauliOp
+    >>> from qiskit.extensions import HamiltonianGate
+    >>> from qiskit import QuantumCircuit
+
+    >>> evol_gate = HamiltonianGate([[0, 1], [1, 0]], 1)
+    >>> evolved_state = QuantumCircuit(1)
+    >>> evolved_state.append(evol_gate, [0])
+
+    >>> print(evolved_state.decompose().decompose())
+       ┌────────────────┐
+    q: ┤ U3(2,-π/2,π/2) ├
+       └────────────────┘
+
+.. raw:: html
+
+    </details>
+
+Expectations
+============
+*Back to* `Contents`_
+
+Expectations are converters which enable the computation of the expectation value of an observable with respect to some state function.
+This functionality can now be found in the Estimator primitive.
+
+Algorithm-Agnostic Expectations
+-------------------------------
+*Back to* `Contents`_
+
+.. list-table::
+   :header-rows: 1
+
+   * - Opflow
+     - Alternative
+
+   * - :class:`~qiskit.opflow.expectations.ExpectationFactory`
+     - No direct replacement. This class was used to create instances of one of the classes listed below.
+
+   * - :class:`~qiskit.opflow.expectations.AerPauliExpectation`
+     - Use ``Estimator`` primitive from |qiskit_aer.primitives|_ with ``approximation=True`` and ``shots=None`` as ``run_options``.
+       See example below.
+
+   * - :class:`~qiskit.opflow.expectations.MatrixExpectation`
+     - Use :class:`~qiskit.primitives.Estimator` primitive from :mod:`qiskit` (if no shots are set, it performs an exact Statevector calculation).
+       See example below.
+
+   * - :class:`~qiskit.opflow.expectations.PauliExpectation`
+     - Use any Estimator primitive (for :class:`qiskit.primitives.Estimator`, set ``shots!=None`` for a shot-based
+       simulation, for |qiskit_aer.primitives.Estimator|_ , this is the default).
+
+
+.. _expect_state:
+
+
+.. raw:: html
+
+    <details>
+    <summary><a><font size="+1">Example 1: Aer Pauli expectation</font></a></summary>
+    <br>
+
+**Opflow**
+
+.. code-block:: python
+
+    >>> from qiskit.opflow import X, Minus, StateFn, AerPauliExpectation, CircuitSampler
+    >>> from qiskit.utils import QuantumInstance
+    >>> from qiskit_aer import Aer
+
+    >>> backend = Aer.get_backend("aer_simulator")
+    >>> q_instance = QuantumInstance(backend)
+
+    >>> sampler = CircuitSampler(q_instance, attach_results=True)
+    >>> expectation = AerPauliExpectation()
+
+    >>> state = Minus
+    >>> operator = 1j * X
+
+    >>> converted_meas = expectation.convert(StateFn(operator, is_measurement=True) @ state)
+    >>> expectation_value = sampler.convert(converted_meas).eval()
+
+    >>> print(expectation_value)
+    -1j
+
+**Alternative**
+
+.. code-block:: python
+
+    >>> from qiskit.quantum_info import SparsePauliOp
+    >>> from qiskit import QuantumCircuit
+    >>> from qiskit_aer.primitives import Estimator as AerEstimator
```

Comment:
```suggestion
    >>> from qiskit_aer.primitives import Estimator
```

Do we really need import `Estimator as AerEstimator`? In this example, there is no clash of Aer's Estimator with Terra's Estimator. I think one great benefits of using the same class name is that it can be changed among different implementations of primitives easily.